.circleci/base-install.sh

```diff
@@ -4,13 +4,10 @@ MODULE=$1
 DIR=$(dirname "$0")
 cd "$DIR/.."
 
-# npm install just enough to run these scripts
-npm i --ignore-scripts --no-optional --only=prod --no-package-lock
```

Comment:
npm bundled with 16 is workspace aware and doesn't like that we're using `workspace:*` ... we can just use yarn, it's nbd